clients/cpp/src/signalrlib/signalrlib.filters

```diff
@@ -0,0 +1,35 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Why signalrlib and not lib or just signalr?

Reply:
actually should be libsignalr for mac and linux